tests/e2e/demo-homepages.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { expect, test } from '@playwright/test';
+
+test.describe('Demo Homepages', () => {
+  const demos = [
+    {
+      url: 'https://spirit-design-system-demo.netlify.app/',
+      package: 'web',
+    },
+    {
+      url: 'https://spirit-design-system-react.netlify.app/',
+      package: 'web-react',
+    },
+  ];
+
+  for (const demo of demos) {
+    test(`test demo homepage ${demo.package}`, async ({ page }) => {
+      await page.goto(demo.url);
+      await expect(page).toHaveScreenshot(`${demo.package}.png`, { fullPage: true, maxDiffPixelRatio: 0.01 });
```

Comment:
`maxDiffPixelRatio` - the visuals slightly differ on GitHub Actions on each run. I think it is due to the font rendering. Locally Docker does it as well but less often.